layers/+lang/go/README.org

```diff
@@ -67,6 +67,7 @@ formatter, set the value of =gofmt-command=, e.g.
 | ~SPC m e b~ | go-play buffer                                            |
 | ~SPC m e r~ | go-play region                                            |
 | ~SPC m e d~ | download go-play snippet                                  |
+| ~SPC m e e~ | run "go run" for the current 'main' package               |
```

Comment:
Thanks for the suggestion. 

Yes, it runs the main() in the file ( which should be marked as package main). This is similar to mer, but the 'r' is taken. Will make the change and resubmit.